### ISSUE TITLE
TFT28 colours not correct?

### ISSUE THREAD

**Issue by flygja** [makerbase-mks/MKS-TFT#390]
I'm using the red icon pack and retro firmware. When I set the font colour to 0xxFFFFFF, I don't see the labels for the buttons. When I set the status font colour to 0xFFFF00 for yellow, I get blue instead. I have attached screenshots and my config file. I'm using firmware 3.0.5

![20200312_202025](https://user-images.githubusercontent.com/61615482/76521163-3b8e9880-649f-11ea-9d20-d91e7e6060f3.jpg)
![20200312_202056](https://user-images.githubusercontent.com/61615482/76521167-3cbfc580-649f-11ea-9df0-e7c8558ffac9.jpg)

[mks-config.txt](https://github.com/makerbase-mks/MKS-TFT/files/4324097/mks-config.txt)

**Comment by ghost** [comment]
RTFM. This displays use BGR not RGB colors. 0x00FFFF for yellow.

**Comment by flygja** [comment]
The manual says yellow is 0xFFFF00 LOL. Still doesn't explain why the button labels are missing. They are missing even when text colour is set to white 0xFFFFFF.

![Annotation 2020-03-16 101745](https://user-images.githubusercontent.com/61615482/76717726-7a5a7200-676f-11ea-9d44-9ff46f991836.png)

**Comment by ghost** [comment]
Nice joke from our friends =))). I can't see cfg_btn_text_offset parameter in MKS-TFT28 default config, like in MKS-TFT3.5 config. Is MKS-TFT28 firmware supports this parameter? If cfg_btn_text_offset value is too high or too low button text becomes invisible.
BTW, I think there is no solution for this problem, because [majurka](https://github.com/majurca/MKS-TFT28-NEW-PICTURES) theme have text embedded in pictures. Pretty sure this is for good reason.

**Comment by flygja** [comment]
Nope, this is not in the config file for TFT28. I think when I change any "background color" option to anything except black, the text labels disappear. If I change it back to black, then they all appear again.
